modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```
@@ -1042,6 +1044,18 @@ private void stopCache(GridCacheAdapter<?, ?> cache, boolean cancel, boolean des
             if (destroy)
                 cache.removeMetrics();
 
+            if (ctx.kernalContext().query().moduleEnabled()) {
+                IgniteInternalFuture<?> rebFut = ctx.kernalContext().query().getIndexing().indexRebuildFuture(ctx);
+
+                if (rebFut != null) {
+                    try {
+                        rebFut.get();
+                    }
+                    catch (IgniteCheckedException ignore) {
```

why do we ignore error here?